matter_server/server/device_controller.py

```diff
@@ -1024,7 +1024,7 @@ async def _check_node_update(
             node_logger.info("No new update found.")
             return None, None
 
-        if "otaUrl" not in update:
+        if "otaUrl" not in update or update["otaUrl"].trim().length == 0:
```

Comment:
```suggestion
        if "otaUrl" not in update or update["otaUrl"].strip().length == 0:
```

Reply:
Actually, this check should not be there :see_no_evil:  I moved it to the `_check_update_version()` function.

Well, I guess we could also drop it here, in case someone adds a local udpate with an empty URL :thinking:

Reply:
Fixed!